src/kafka/apache.rs

```diff
@@ -0,0 +1,335 @@
+use std::{borrow::Cow, collections::HashMap};
+use testcontainers::{
+    core::{ContainerPort, ContainerState, ExecCommand, WaitFor},
+    Image,
+};
+
+const KAFKA_NATIVE_IMAGE_NAME: &str = "apache/kafka-native";
+const KAFKA_IMAGE_NAME: &str = "apache/kafka";
+const TAG: &str = "latest";
```

Comment:
Oops, missed that part.
We do not use the latest version because it does not guarantee image stability. Users can always override it to the latest version if they want, but this easily leads to instability and the implementation can quickly become outdated.